src/plugins/vis_types/timeseries/public/application/components/aggs/top_hit.js

```diff
@@ -199,14 +200,10 @@ const TopHitAggUi = (props) => {
               <FormattedMessage id="visTypeTimeseries.topHit.sizeLabel" defaultMessage="Size" />
             }
           >
-            {/*
-              EUITODO: The following input couldn't be converted to EUI because of type mis-match.
-              Should it be text or number?
-            */}
-            <input
-              className="tvbAgg__input"
+            <EuiFieldNumber
+              onChange={handleNumberChange('size', { isClearable: true })}
```

Comment:
Please see how that feature was implemented in `Percentile Rank` agg. I like the approach with using the placeholder. 

Please look at the next screen, there are at least 2 issues =)

https://user-images.githubusercontent.com/20072247/172588990-f1b6e53e-bb92-4911-af5a-fb4800af3802.mov

Reply:
done